moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummary.js

```diff
@@ -162,6 +162,15 @@ const ProjectSummary = ({ loading, error, data, refetch }) => {
               classes={classes}
               snackbarHandle={snackbarHandle}
             />
+            {data.moped_project[0]?.parent_project_id && (
+              <ProjectSummaryParentProjectLink
+                projectId={projectId}
+                data={data}
+                refetch={refetch}
+                classes={classes}
+                snackbarHandle={snackbarHandle}
```

Comment:
It doesn't seem like ProjectSummaryParentProjectLink uses some of these props if I'm looking at it right.